config/jobs/kubernetes/sig-node/crio.yaml

```
@@ -512,3 +512,54 @@ periodics:
     testgrid-alert-email: kubernetes-sig-node-test-failures@googlegroups.com
     description: "OWNER: sig-node; runs NodeConformance e2e tests with crio master,
       cgroup v1 and the evented pleg feature enabled"
+- name: ci-crio-CDI-device-plugins
```

```suggestion
- name: ci-crio-cdi-device-plugins
```
CI seems to not like this